app/controllers/navigation/Navigator.scala

```diff
@@ -40,12 +40,12 @@ import uk.gov.hmrc.http.HeaderCarrier
 
 class Navigator @Inject()(appConfig: AppConfig, auditService: AuditService) {
 
-  def continueTo(mode: Mode, factory: Mode => Call)(implicit req: JourneyRequest[AnyContent], hc: HeaderCarrier): Result =
+  def continueTo(mode: Mode, factory: Mode => Call, isErrorFixInProgress: Boolean = false)(
```

Comment:
Would it make sense to have this `ErrorFixInProgress` as a new `Mode` ?

Reply:
In my opinion the last parameter is not necessary. In every place you pass it, you are doing the same comparision. Why not doing it inside `continueTo` method? You have access to the `mode` field so you can preserve the logic, but you don't clutter method's interface.
Unless there are some other non-obvious factors that I am not aware of.

Reply:
Hmm, that can be a solution too, but with new mode you'll need to have this logic inside the controller, so I'll come back to previous solution

Reply:
I agree with second comment, but I think that it won't handle the logic as it should. I'll check it right now

Reply:
Yeah, we need only pass this elemnt in scenario where we're considering different behaviour than normal one. This is the reason that this can't leave inside the Navigator

Reply:
I think new mdoe won't help, still there need to be logic for it :(